src/main/java/net/openhft/chronicle/wire/YamlWire.java

```diff
@@ -1779,12 +1774,7 @@ public <T> boolean sequence(@NotNull List<T> list, @NotNull List<T> buffer, @Not
                 int minIndent = yt.secondTopContext().indent;
                 yt.next(Integer.MAX_VALUE);
 
-                while (hasNextSequenceItem()) {
-                    if (buffer.size() <= list.size())
-                        buffer.add(bufferAdd.get());
-                    Object using = buffer.get(list.size());
-                    list.add((T) valueIn.object(using, using.getClass()));
-                }
+                reader0.accept(this, list, buffer, bufferAdd);
```

Comment:
this code did not call `if (t instanceof Resettable) ((Resettable) t).reset();` before, unlike other implementations in other Wires, so that is a nice by-product